src/components/screens/InfoCard.js

```diff
@@ -40,7 +44,11 @@ const contactView = (contact) => (
     {!!contact.email && (
       <InfoBox>
         <Icon xml={mail(colors.primary)} style={styles.margin} />
-        <TouchableOpacity onPress={() => openLink(`mailto:${contact.email}`)}>
+        <TouchableOpacity
+          onPress={() => openLink(`mailto:${contact.email}`)}
+          accessibilityLabel="E-mail-Adresse"
+          accessibilityHint="Navigieren zur E-mail-App"
```

Comment:
lets go with "Wechselt zu E-Mail-App" here, "navigieren" indicates moving inside of the app.